README.md

```diff
@@ -1,3 +1,13 @@
+> “In this way you must understand how laughable it is to say, ‘Tell me what to do!’ 
+> What advice could I possibly give? 
+> No, a far better request is, 
+> ‘Train my mind to adapt to any circumstance.’ […] 
+> In this way, if circumstances take you off script […] 
+> you won’t be desperate for new prompting.“
+
+> – Epictetus, _Discourses_, 2.2.20b-1;24b-25a (~108 CE)
```

Comment:
Hmm, this ain't adding new lines according to preview 🤔

Reply:
![image](https://user-images.githubusercontent.com/1889973/44905419-e7cbbc80-ad09-11e8-8584-953314ed49cd.png)

Reply:
Apparently we can add `</br>` tag at the end if we want to split them line by line:
```
> line 1 <br>
> line 2
```

Reply:
I wanted to keep it fairly short on the site. I think this is ok